tests/openai/mock_openai.py

```diff
@@ -15,9 +15,25 @@ def health():
     return {"status": "healthy"}
 
 
+class TextContentBlock(BaseModel):
+    type: Literal["text"]
+    text: str
+
+
+class ImageUrl(BaseModel):
+    url: str
+
+
+class ImageContentBlock(BaseModel):
+    type: Literal["image_url"]
+    image_url: ImageUrl
+
+
```

Comment:
do we need audio content block? 

<img width="543" alt="Screenshot 2024-12-12 at 9 35 44 AM" src="https://github.com/user-attachments/assets/2bf3718b-e045-4e82-b522-f6a8aa960944" />

Reply:
That's a good point. Since we're using the SDK, this should theoretically be supported, right? :)

Reply:
Let me check the open SDK for what's supported.

Reply:
Added audio. We don't need it now, but we might in the future.